dataset/dictionary/characters.json5

```
@@ -207,87 +239,105 @@
   {
     en: "Diona",
     ja: "ディオナ",
+    zh: "迪奥娜",
     notes: "読みは「ダイオナ」",
     tags: [ "mondstadt", "character-main" ],
   },
   {
     en: "Barbara Pegg",
     ja: "バーバラ・ペッチ / バーバラ・ペイジ",
+    zh: "芭芭拉·佩奇",
     tags: [ "mondstadt", "character-main" ],
     notes: "姓については表記揺れあり。英語版の発音は「バーバラ・ペッグ」",
```

Let's add your comment in `notes` section.
The added text means "In China, her last name is not considered 'Pegg' but 'Gunnhildr'."
Since "Barbara Pegg" should sound strange for Chinese players, I will remove Pegg and add references to Jean Gunnhildr & Seamus Pegg when Chinese UI is ready.
```suggestion
    notes: "姓については表記揺れあり。英語版の発音は「バーバラ・ペッグ」。中国では姓は「ペッチ」「ペイジ」ではなく「グンヒルド」とする説が有力。",
```